src/app/shared/my-journey-ongoing-card/my-journey-ongoing-card.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="ongoing-card" (click)="navigateproject(data._id)">
+  <div class="ongoing-head">
+    <div>
+      @if (data.endDate) {
+        <p>{{"DUE_DATE" | translate}}: {{ data.endDate | date:'dd/MM/yy' }}</p>
+      }
+    </div>
+    <span>
+      <button>
+        <mat-icon>arrow_forward</mat-icon>
+      </button>
+    </span>
+  </div>
+  <div class="ongoing-title">
+    <p>{{data.title | shortUrl:45}}</p>
+  </div>
+  <div class="progress-bar">
+    <span class="progress-text">{{ calculateProgress(data) }}%</span>
```

Comment:
In this line and line 22 donot call the function in the html
Since its a card calculate it and store it in a value during ngOnInit() and use that value here

Reply:
done